docs/setup/implementation_setup_patch.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Patch implementation
+nav_order: 1
+parent: Implementation setup
+---
+
+# Patch implementation setup
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+- TOC
+{:toc}
+
+The `patch` tag describes patch framework-specific setup.
+Nested tags include `patch.parameter` for parameters.
+
+```xml
+<patch>
+    <patch.parameter id="[ID]" value="[VALUE]" scale="[SCALE]" />
+    ...
+</patch>
+```
+
+## Implementation parameters
+
+The `patch.parameter` tag defines patch implementation parameters.
+Unless modified, default values are used for all parameters.
+Defaults are listed in [parameter.patch.xml](https://github.com/bagherilab/ARCADE/blob/main/src/arcade/patch/parameter.patch.xml).
+
+- Either or both `value` and `scale` attributes can be applied, with `value` applied first
+- Only numeric parameters can be modified using `scale`
+
+| ATTRIBUTE | DESCRIPTION                               |
+| --------- | ----------------------------------------- |
+| `id`      | Parameter name                            |
+| `value`   | New parameter value                       |
+| `scale`   | Scaling factor applied to parameter value |
```

Comment:
Should scale be included here, as the examples are strings?

Reply:
@cainja Yeah, there are two options, and I went with the first one but can switch to the second if it is more clear:

1. Current `patch` parameters are only strings, so `scale` does not apply. But, a numeric parameter might be added in the future. There is a note that "Only numeric parameters can be modified using `scale`"
2. There are no numeric parameters for `patch` so drop both the `scale` in the table and the note about numeric parameters. If a numeric parameter is added, then remember to update this doc

I think right now trying to scale a string will throw an error, but we should probably just catch that case and log a warning.

Reply:
I think option 1 is fine for now! (Although I am struggling to find a situation where we'd like to include a numeric value)